# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 16a3de29e4e6
# Full Hash: 16a3de29e4e69175f5a7c914ffa9de63b8683f2a
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2023-09-28 21:51:27
# Regressor Bug: 1731541
# File Overlap Count: 1
# Description:
#   Bug 1731541 - When line-clamp is in effect, make text-wrap:balance consider only the lines up to the clamp limit. r=emilio
#   
#   This corresponds to how Chrome behaves, and passes the test they included in WPT.
#   
#   It's unclear to me whether this behavior actually follows from the current spec
# ==============================================================================

diff -r 225642ff1de8 -r 16a3de29e4e6 modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Thu Sep 28 17:28:19 2023 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Thu Sep 28 17:28:20 2023 +0000
@@ -8775,6 +8775,11 @@
   value: 10
   mirror: always
 
+- name: layout.css.text-wrap-balance-after-clamp.enabled
+  type: bool
+  value: true
+  mirror: always
+
 # Support for the css Zoom property.
 - name: layout.css.zoom.enabled
   type: RelaxedAtomicBool